modules/local/busco-auto-lineage.nf

```diff
@@ -0,0 +1,107 @@
+process BUSCO {
+    tag "$meta.id"
+    label 'process_large'
+
+    conda (params.enable_conda ? "bioconda::busco=5.4.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/busco:5.4.3--pyhdfd78af_0':
+        'quay.io/biocontainers/busco:5.4.3--pyhdfd78af_0' }"
+        
+    publishDir(
+        path: "${params.outdir}/BUSCO",
+        mode: 'copy',
+        saveAs: { fn -> fn.substring(fn.lastIndexOf('/')+1) },
```

Comment:
If I'm being entirely honest I only really know what two parts of it does. It's just specifying the name of the the output directory. Now, the "mode" and "saveAs" part I'm not sure. I found this online somewhere and it seemed to work, so I didn't fix what wasn't broken - though it might be dumb still?

For instance, I'm wondering if the "mode: 'copy'" part means that it's duplicating outputs somehow which might be a bad idea? I'm not sure. And I really am not sure about that "saveAs" bit.....